magento/pwa-studio
# [Doc] Targetables

## Description

Create documentation for Targetables, this includes:

- A brief explainer
- An overview for how to use it
- Reference documentation with code examples 

## Related Issue

Closes PWA-796

## Acceptance

Any core developer

### Verification Stakeholders

Any core developer

### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` project: `cd pwa-devdocs`
2. Run the HTML preview server: `yarn develop`
3. Verify site builds
4. Navigate to `/pwa-buildpack/extensibility-framework/`
5. Verify Targetables section exists on the page with accurate information
6. Navigate to `/tutorials/targetables/`
7. Verify page exists, contains accurate information and listed on the left navigation
8. Navigate to `/pwa-buildpack/reference/targetables/TargetableSet/`
9. Verify **Targetables** section listed on the left navigation with working links to individual reference doc pages

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
